boards/posix/native_sim/doc/index.rst

```diff
@@ -433,10 +433,20 @@ The following peripherals are currently provided with this board:
 
 .. _nsim_per_flash_simu:
 
-**Flash driver**
-  A flash driver is provided that accesses all flash data through a binary file
-  on the host file system. The behavior of the flash device can be configured
-  through the native_sim board devicetree or Kconfig settings under
+**EEPROM simulator**
+  The EEPROM simulator can also be used in the native targets.
+  In these, you have the added feature of keeping the EEPROM content
+  on a file on the host filesystem.
+  By default this is kept in the file :file:`eeprom.bin` in the current
+  working directory, but you can select the location of this file and its name with the
+  command line parameter ``--eeprom``.
```

Comment:
nit: your line length wrapping seems a bit random :-) not that the other files are consistent but I'd be cool if you could pick one and keep it for the whole block at least.